includes/libraries/parsecsv.lib.php

```diff
@@ -0,0 +1,1163 @@
+<?php
+
+class parseCSV {
```

Comment:
If we're patching this library, there's a newer one available: https://github.com/parsecsv/parsecsv-for-php/releases